Core/include/Acts/Seeding/HoughVectors.hpp

```diff
@@ -28,12 +28,12 @@ class MultiIndexedVector2D {
 
  public:
   MultiIndexedVector2D() : m_d1(0), m_d2(0) {
-    m_logger = Acts::getDefaultLogger("HoughVectors", Acts::Logging::ERROR);
+    m_logger = Acts::getDefaultLogger("HoughVectors", Acts::Logging::WARNING);
```

Comment:
Is this a temporary change?

Reply:
I am not sure, I had the failure of the CI for the unit test for the THRESHOL_LOG_LEVEL before. That's why I changed it
Maybe @goblirsc could give an opinion?

Reply:
You should be able to ignore the logging-warning in the unit test with a scoped threshold. Maybe have a look at this PR:
- https://github.com/acts-project/acts/pull/3290

Reply:
@AJPfleger is right, but this really should only be done right where you expect the error output because if a failure condition that you're testing. Otherwise, this can hide the kinds of accidental error messages that we're trying to avoid.

Reply:
Thank you @paulgessinger and @AJPfleger  !